23:29:45,737 WARN  eu.stratosphere.nephele.configuration.GlobalConfiguration     - Cannot load configuration: unknown element plugins
23:29:45,774 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Reading location of job manager from configuration
23:29:45,784 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Determined address of job manager to be cloud-11.dima.tu-berlin.de/130.149.21.15:6002
23:29:45,792 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Announcing connection information cloud-31 to job manager
23:29:45,857 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server listener on 6122: starting
23:29:45,857 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server Responder: starting
23:29:45,858 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server handler 0 on 6122: starting
23:29:46,978 INFO  eu.stratosphere.nephele.taskmanager.bufferprovider.GlobalBufferPool  - Initialized global buffer pool with 32768 buffers with a size 32768 bytes each
23:29:46,997 INFO  eu.stratosphere.nephele.taskmanager.bytebuffered.ByteBufferedChannelManager  - Initialized byte buffered channel manager with sender-side spilling disabled
23:29:47,006 INFO  eu.stratosphere.nephele.instance.HardwareDescriptionFactory   - Found Tenured Gen pool (max: 24051843072, used: 0)
23:29:47,006 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Initializing memory manager with 22528 megabytes of memory
23:30:18,645 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Build Triads (18/20)
23:30:18,648 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Build Triads (18/20)
23:30:19,946 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (137/160)
23:30:19,947 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (137/160)
23:30:19,949 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (137/160)
23:30:19,950 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (137/160)
23:30:21,454 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (139/160)
23:30:21,454 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (139/160)
23:30:21,456 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (139/160)
23:30:21,457 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (139/160)
23:30:22,067 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (138/160)
23:30:22,068 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (138/160)
23:30:22,070 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (138/160)
23:30:22,071 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (138/160)
23:30:22,363 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (141/160)
23:30:22,364 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (141/160)
23:30:22,365 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (141/160)
23:30:22,366 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (141/160)
23:30:22,671 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (143/160)
23:30:22,672 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (143/160)
23:30:22,673 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (143/160)
23:30:22,674 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (143/160)
23:30:23,543 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (140/160)
23:30:23,543 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (140/160)
23:30:23,545 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (140/160)
23:30:23,545 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (140/160)
23:30:23,582 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (144/160)
23:30:23,583 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (144/160)
23:30:23,583 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (144/160)
23:30:23,584 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (144/160)
23:30:25,199 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (142/160)
23:30:25,200 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (142/160)
23:30:25,201 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (142/160)
23:30:25,202 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (142/160)
23:31:08,154 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Finished PACT code.: Build Triads (18/20)
23:31:08,155 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from RUNNING to FINISHING for task Build Triads (18/20)
23:35:10,975 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_4894719964700498764_1565java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:34445 remote=/130.149.21.33:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:35:10,975 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_4894719964700498764_1565 bad datanode[0] 130.149.21.33:45020
23:35:10,976 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_4894719964700498764_1565 in pipeline 130.149.21.33:45020, 130.149.21.26:45020, 130.149.21.16:45020: bad datanode 130.149.21.33:45020
23:35:11,042 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1336474425465320789_1565java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:34449 remote=/130.149.21.33:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:35:11,043 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1336474425465320789_1565 bad datanode[0] 130.149.21.33:45020
23:35:11,044 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1336474425465320789_1565 in pipeline 130.149.21.33:45020, 130.149.21.30:45020, 130.149.21.26:45020: bad datanode 130.149.21.33:45020
23:35:11,155 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-8206134601034160365_1565java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:34455 remote=/130.149.21.33:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:35:11,155 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8206134601034160365_1565 bad datanode[0] 130.149.21.33:45020
23:35:11,156 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8206134601034160365_1565 in pipeline 130.149.21.33:45020, 130.149.21.30:45020, 130.149.21.2:45020: bad datanode 130.149.21.33:45020
23:35:11,692 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-46998981425306157_1565java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:34446 remote=/130.149.21.33:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:35:11,693 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-46998981425306157_1565 bad datanode[0] 130.149.21.33:45020
23:35:11,694 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-46998981425306157_1565 in pipeline 130.149.21.33:45020, 130.149.21.3:45020, 130.149.21.18:45020: bad datanode 130.149.21.33:45020
23:35:11,732 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_5281546100309610389_1565java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:34451 remote=/130.149.21.33:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:35:11,733 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5281546100309610389_1565 bad datanode[0] 130.149.21.33:45020
23:35:11,733 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5281546100309610389_1565 in pipeline 130.149.21.33:45020, 130.149.21.19:45020, 130.149.21.61:45020: bad datanode 130.149.21.33:45020
23:35:11,827 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-832419634506320424_1565java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:34460 remote=/130.149.21.33:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:35:11,828 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-832419634506320424_1565 bad datanode[0] 130.149.21.33:45020
23:35:11,829 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-832419634506320424_1565 in pipeline 130.149.21.33:45020, 130.149.21.25:45020, 130.149.21.22:45020: bad datanode 130.149.21.33:45020
23:35:11,964 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-8308670571066758511_1565java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:34453 remote=/130.149.21.33:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:35:11,965 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8308670571066758511_1565 bad datanode[0] 130.149.21.33:45020
23:35:11,966 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8308670571066758511_1565 in pipeline 130.149.21.33:45020, 130.149.21.3:45020, 130.149.21.30:45020: bad datanode 130.149.21.33:45020
23:36:17,146 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-1336474425465320789_1668java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:37064 remote=/130.149.21.30:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:36:17,147 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1336474425465320789_1668 bad datanode[0] 130.149.21.30:45020
23:36:17,148 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-1336474425465320789_1668 in pipeline 130.149.21.30:45020, 130.149.21.26:45020: bad datanode 130.149.21.30:45020
23:36:17,254 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-8206134601034160365_1669java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:37066 remote=/130.149.21.30:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:36:17,255 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8206134601034160365_1669 bad datanode[0] 130.149.21.30:45020
23:36:17,255 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8206134601034160365_1669 in pipeline 130.149.21.30:45020, 130.149.21.2:45020: bad datanode 130.149.21.30:45020
23:36:17,795 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-46998981425306157_1670java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:45680 remote=/130.149.21.3:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:36:17,795 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_5281546100309610389_1671java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:32874 remote=/130.149.21.19:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:36:17,796 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-46998981425306157_1670 bad datanode[0] 130.149.21.3:45020
23:36:17,797 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5281546100309610389_1671 bad datanode[0] 130.149.21.19:45020
23:36:17,797 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-46998981425306157_1670 in pipeline 130.149.21.3:45020, 130.149.21.18:45020: bad datanode 130.149.21.3:45020
23:36:17,798 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5281546100309610389_1671 in pipeline 130.149.21.19:45020, 130.149.21.61:45020: bad datanode 130.149.21.19:45020
23:36:17,928 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-832419634506320424_1672java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:39927 remote=/130.149.21.25:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:36:17,929 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-832419634506320424_1672 bad datanode[0] 130.149.21.25:45020
23:36:17,929 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-832419634506320424_1672 in pipeline 130.149.21.25:45020, 130.149.21.22:45020: bad datanode 130.149.21.25:45020
23:36:18,066 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-8308670571066758511_1673java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:45686 remote=/130.149.21.3:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:36:18,067 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8308670571066758511_1673 bad datanode[0] 130.149.21.3:45020
23:36:18,068 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8308670571066758511_1673 in pipeline 130.149.21.3:45020, 130.149.21.30:45020: bad datanode 130.149.21.3:45020
23:36:42,090 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_4894719964700498764_1667java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.33:49698 remote=/130.149.21.26:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

23:36:42,091 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_4894719964700498764_1667 bad datanode[0] 130.149.21.26:45020
23:36:42,091 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_4894719964700498764_1667 in pipeline 130.149.21.26:45020, 130.149.21.16:45020: bad datanode 130.149.21.26:45020
